### PR TITLE
[Network] az network private-endpoint-connection: Support more services for private link

### PR DESCRIPTION
**Pull request by MyronFanQiu**
**Description**  

**Testing Guide**  

**History Notes**  

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.  
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/azure-cli/azure/cli/command_modules/network/private_link_resource_and_endpoint_connections/custom.py [hunk 1]
```diff
@@ -28,6 +28,14 @@ def register_providers():
     _register_one_provider('Microsoft.EventGrid/domains', '2020-04-01-preview', True)
     _register_one_provider('Microsoft.SignalRService/signalr', '2020-05-01', False)
     _register_one_provider('Microsoft.Network/applicationGateways', '2020-05-01', True)
+    _register_one_provider('Microsoft.AppConfiguration/configurationStores', '2019-11-01-preview', True)
+    # Don't have tests right now
+    _register_one_provider('Microsoft.AppConfiguration/configurationStores', '2019-11-01-preview', True)
```
Comment (dingmeng-xue) [inline src/azure-cli/azure/cli/command_modules/network/private_link_resource_and_endpoint_connections/custom.py:33]: why is it repeated?

**Comment by yonzhan** [comment]
private link

File: src/azure-cli/azure/cli/command_modules/network/private_link_resource_and_endpoint_connections/custom.py (hunk 1 above)
Comment (MyronFanQiu) [reply src/azure-cli/azure/cli/command_modules/network/private_link_resource_and_endpoint_connections/custom.py:32]: good catch!. My mistake. Removed.

**Comment by yonzhan** [comment]
@dingmeng-xue Thanks for the approval.

File: src/azure-cli/azure/cli/command_modules/network/private_link_resource_and_endpoint_connections/custom.py [hunk 2]
```diff
@@ -28,6 +28,13 @@ def register_providers():
     _register_one_provider('Microsoft.EventGrid/domains', '2020-04-01-preview', True)
     _register_one_provider('Microsoft.SignalRService/signalr', '2020-05-01', False)
     _register_one_provider('Microsoft.Network/applicationGateways', '2020-05-01', True)
+    # Don't have tests right now
+    _register_one_provider('Microsoft.AppConfiguration/configurationStores', '2019-11-01-preview', True)
```
Comment (avanigupta) [inline src/azure-cli/azure/cli/command_modules/network/private_link_resource_and_endpoint_connections/custom.py:32]: Can you change API version to 2020-06-01?

**Comment by MyronFanQiu** [comment]
Close this PR first since it should be done right now.

**Pull request closed without merging**